.circleci/config.yml

```diff
@@ -23,6 +23,40 @@ commands:
       - run: git config --global user.email "dev+ci@revenuecat.com"
       - run: git config --global user.name "RevenueCat CI"
 
+  trust-github-key:
+    steps:
+      - run:
+          name: Trust GitHub key
+          command: |
+            for ip in $(dig @8.8.8.8 github.com +short); \
+            do ssh-keyscan github.com,$ip; \
+            ssh-keyscan $ip; \
+            done 2>/dev/null >> ~/.ssh/known_hosts
+  install-rubydocker-dependencies:
+    steps:
+      - restore_cache:
+          keys:
+            - v1-rubydocker-gem-cache-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Bundle install
+          command: bundle install --clean --path vendor/bundle
+      - save_cache:
+          key: v1-rubydocker-gem-cache-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+  install-macdocker-dependencies:
```

Comment:
Not important but I don't think the max executor is a docker. I might be wrong 😅